src/main/java/com/singh/astha/notification/service/exceptions/handler/GlobalExceptionHandler.java

```diff
@@ -29,8 +29,7 @@ public ResponseEntity<ResponseWrapper<Object>> handle(
             ResponseException exception
     ) {
         // FIXME: Update the Response Exception
-        ErrorResponse errorResponse = ErrorResponse.builder().build();
-        errorResponse.setMessage(exception.getMessage());
+        ErrorResponse errorResponse = ErrorResponse.from(exception.getMessage());
```

Comment:
Remove this.

Instead of this include `private final List<ErrorResponse> errorResponses;` in the ResponseException class.

And then use it directly here.